pkg/volume/util.go

```diff
@@ -91,7 +91,7 @@ func internalRecycleVolumeByWatchingPodUntilCompletion(pvName string, pod *v1.Po
 			if deleteErr != nil {
 				return fmt.Errorf("failed to delete old recycler pod %s/%s: %s", pod.Namespace, pod.Name, deleteErr)
 			}
-			// Recycler will try again and the old pod will be hopefuly deleted
+			// Recycler will try again and the old pod will be hopefully deleted
```

Comment:
This has been fixed in https://github.com/kubernetes/kubernetes/pull/59464 - https://github.com/kubernetes/kubernetes/pull/59464/files#diff-e01789dba347ac8adc158d1bcde5cc0f. Closing.

/close